snapshot.yaml

```diff
@@ -1,4 +1,4 @@
-# 20240229_0932
+# 20240304_2200
```

Comment:
```suggestion
```

Reply:
todo: https://github.com/Samsung/CredSweeper/pull/520 - after this it will be not necessary